docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Learn how to configure SAML authentication in Grafana's UI.
+menuTitle: Configure SAML authentication in UI
+title: Configure SAML authentication in Grafana's UI
+weight: 1150
+---
+
+# Configure SAML authentication in Grafana
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise/" >}}) version 10.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
+
+You can configure SAML authentication in Grafana either through the UI or the Grafana config file.
```

Comment:
```suggestion
You can configure SAML authentication in Grafana through the user interface (UI) or the Grafana configuration file.
```

Reply:
Pls correct the path if I have that incorrect, and the name of the topic SAML config topic if you have changed that.